packages/terra-list/tests/wdio/selectable-list-spec.js

```diff
@@ -0,0 +1,139 @@
+      browser.waitForExist('ul');
+      browser.click('ul li:nth-child(1)');
+    });
+
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-list-item-focus-background-color': '#fdebeb',
+        '--terra-list-item-focus-box-shadow': '5px 10px #dddddd',
+        '--terra-list-item-focus-outline': '1px dashed #000000',
+      },
+    });
+  });
+
+  describe('Selectable List Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/default-selectable-list'));
+
+    Terra.should.matchScreenshot();
+
+    it('selects the first item', () => {
+      browser.keys(['Tab', 'Space']);
+    });
+    Terra.should.matchScreenshot('first item selected');
+
+    it('selects the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('second item selected');
+
+    it('selects the third item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('third item selected');
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Disable Unselected', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/disable-unselected-selectable-list'));
+
+    it('selects the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('first item selected');
+
+    it('selects the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('second item selected');
+
+    it('attempts to select the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('third item disabled');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Items Divided', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/items-divided-selectable-list'));
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Non Selectable Item', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/non-selectable-item-selectable-list'));
+
+    Terra.should.matchScreenshot();
+    it('attempts to select non-selectable item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List On Change', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/on-change-selectable-list'));
+
+    it('selects the first item', () => {
+      browser.keys(['Tab', 'Space']);
+    });
+    Terra.should.matchScreenshot('first item selected');
+
+    it('selects the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('second item selected');
+
+    it('attemps to select the third item', () => {
```

Comment:
This example should be able to select the third item just fine: https://terra-core-deployed-pr-1576.herokuapp.com/#/tests/terra-list/list/selectable-list/on-change-selectable-list

attemps to selects && then update the click to click on the third item in the list.

Reply:
Oops, I will rename that test case. The original Nightwatch test presses the first item again.
```javascript
'Triggers onChange for a selectable list': (browser) => {
  browser.url(`${browser.launchUrl}/#/raw/tests/terra-list/list/selectable-list/on-change-selectable-list`);
  browser.sendKeys('ul li:nth-child(1)', browser.Keys.SPACE);
  browser.assert.containsText('#current-index', '0');

  browser.sendKeys('ul li:nth-child(2)', browser.Keys.ENTER);
  browser.assert.containsText('#current-index', '1');

  browser.click('ul li:nth-child(1)');
  browser.assert.containsText('#current-index', '0');
},
```